src/Templates/Microsoft.NetStandard.FSharp.ProjectTemplates/ClassLibrary.vstemplate

```diff
@@ -3,7 +3,7 @@
   <TemplateData>
     <Name Package="{860A27C0-B665-47F3-BC12-637E16A1050A}" ID="5"/>
     <Description Package="{860A27C0-B665-47F3-BC12-637E16A1050A}" ID="6"/>
-    <Icon Package="{FAE04EC1-301F-11d3-BF4B-00C04F79EFBC}" ID="4547" />
+    <Icon Package="{91A04A73-4F2C-4E7C-AD38-C1A68E7DA05C}" ID="4018" />
```

Comment:
If 4002 is the class library icon and 4018 is the test icon, this ID should be 4002, right?

Reply:
yes it should.  Good eyes, thank you.